File: src/kyu_7/sum_of_numbers_from_0_to_n/js/solution.js
var SequenceSum = (function() {
    function SequenceSum() {}

    SequenceSum.showSequence = function(count) {

        if (count === 0) { return count + "=" + 0; }
        if (count < 0) { return count + "<" + 0; }
        let expression = "";
        let result = 0;
        for (let i = 0; i <= count; i++) {
            expression += i + "+";
            result += i;
        }
        return expression.substring(0, expression.length - 1) + " = " + result;
    };

    return SequenceSum;

})();

File: src/kyu_7/sum_of_numbers_from_0_to_n/csharp/solution.cs
public class SequenceSum
{
  public static string ShowSequence(int n)
  {
        if (n == 0) { return n + "=" + 0; }
        if (n < 0) { return n + "<" + 0; }
        string expression = "";
        int result = 0;
        for (int i = 0; i <= n; i++) {
            expression += i + "+";
            result += i;
        }
        return expression.Substring(0, expression.Length - 1) + " = " + result;
  }
}